web/models.py

```
@@ -2608,11 +2611,7 @@ class Meta:
                 name="unique_user_feature_vote",
                 condition=models.Q(user__isnull=False),
             ),
-            models.UniqueConstraint(
-                fields=["feature_id", "ip_address"],
-                name="unique_ip_feature_vote",
-                condition=models.Q(ip_address__isnull=False),
-            ),
+            # Remove the ip_address constraint since encrypted fields can't be used in constraints
         ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)